src/rate.rs

```diff
@@ -500,19 +500,39 @@ impl RCState {
     // Start with a buffer fullness and fullness target of 50%.
     let reservoir_target = (reservoir_max + 1) >> 1;
     // Pick exponents and initial scales for quantizer selection.
-    //
+    let ibpp = npixels/bits_per_frame;
```

Comment:
You should update the comment below this, since its mention of a single bucket is no longer accurate.